src/app/components/OpenChannelModal/index.tsx

```diff
@@ -266,6 +277,8 @@ class OpenChannelModal extends React.Component<Props, State> {
         address,
         capacity,
         isPrivate,
+        // only send non-zero fee
+        fee: fee ? fee.toString() : undefined,
```

Comment:
Does `handleFeeSubmit` in `ChainSend` need the same consideration here? Looks like it could send 0 right now.